### PR TITLE
jpegoptim: add page

### PR DESCRIPTION
**Pull request by sbrl**
A wild PR appeared! This time, it's `jpegoptim`.

### CODE REVIEW

File: pages/common/jpegoptim.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
+
+- Optimise a set of jpeg images:
+
+`jpegoptim {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Optimise jpeg images, stripping all non-essential information:
```
Comment (waldyrious) [inline pages/common/jpegoptim.md]: It's unclear what is the difference between this example and the first. What does "non-essential information" mean? EXIF data? Something else?

File: pages/common/jpegoptim.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
```
Comment (waldyrious) [inline pages/common/jpegoptim.md]: In the descriptions, I'd use the capitalized form (JPEG). But that's just a preference, feel free to ignore.

File: pages/common/jpegoptim.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`jpegoptim {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Optimise jpeg images, stripping all non-essential information:
+
+`jpegoptim --strip-all {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be progressive:
+
+`jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
```
Comment (waldyrious) [inline pages/common/jpegoptim.md]: Why is there an "all" in all-progressive? Is there an option where not all of the images are made progressive?

File: pages/common/jpegoptim.md (hunk 1 above)
Comment (agnivade) [inline pages/common/jpegoptim.md]: Yea, I think we should clarify what sort of optimisations are applied by default, and what does `--strip-all` do.

File: pages/common/jpegoptim.md (hunk 2 above)
Comment (sbrl) [inline pages/common/jpegoptim.md]: Eh, I don't mind either way. Personally, I'd use lowercase, but the `optipng` page uses uppercase so I'll change it that :-)

File: pages/common/jpegoptim.md (hunk 3 above)
Comment (sbrl) [inline pages/common/jpegoptim.md]: I'm not sure! That's the option listed in the man page, anyhow :P

File: pages/common/jpegoptim.md (hunk 1 above)
Comment (sbrl) [inline pages/common/jpegoptim.md]: Yeah, it's EXIF data, but also some extra proprietary markers used by Adobe Photoshop, apparently.

**Comment by sbrl** [comment]
Thanks for the suggestions! :smiley_cat: How's that/

**Pull request merged**